pkg/manager/impl/execution_manager.go

```diff
@@ -1188,6 +1188,9 @@ func (m *ExecutionManager) CreateWorkflowEvent(ctx context.Context, request admi
 		m.systemMetrics.ActiveExecutions.Dec()
 		m.systemMetrics.ExecutionsTerminated.Inc()
 		go m.emitOverallWorkflowExecutionTime(executionModel, request.Event.OccurredAt)
+		if request.Event.GetOutputData() != nil {
```

Comment:
Should we check/validate the size and fail the request if it's bigger than "maxDataSize" used later when retrieving executions?

Reply:
oh you mean and offload the outputs at this time @EngHabu

Reply:
ping @EngHabu

Reply:
Sorry just saw this.. Either offload or fail the request...